addons/web/static/src/js/views/pivot/pivot_model.js

```diff
@@ -335,7 +337,7 @@ var PivotModel = AbstractModel.extend({
             tree = this.colGroupTree;
             group = this._findGroup(this.colGroupTree, groupId[1]);
         }
-        group.directSubTrees = {};
+        group.directSubTrees.clear();
```

Comment:
are you sure 'directSubTrees' always exists here?

Reply:
If the group exists (which should be the case), it is by construction in colGroupTree or rowGroupTree and at each level,
directSubTrees is a Map instance. To summarize, if group is defined, yes. Else we would have an error 'directSubTrees of undefined'. Do you expect this to happen?

Reply:
A concurency issue?

Reply:
No, it's fine then.

Reply:
Can you robodoo r+ then or is there something else? Thank you